pages/about/code-of-conduct.md

```diff
@@ -34,3 +34,13 @@ $(document).ready(function(){
 
 });
 </script>
+
+<p>
+    The current Code of Conduct Committee consists of:
+    <ul>
+        <li> Lezlie Espana</li>
```

Comment:
For Lezlie, should last name be spelled "España" ?

Reply:
Yes, that's correct, thank you!